frontend/src/components/employees/index.tsx

```diff
@@ -139,6 +141,25 @@ const addButton = (
 const Employees = () => {
   const staffList = useSelector(selectStaff);
   const dispatch = useDispatch();
+  const [isDeleteModalOpen, setDeleteModalState] = useState(false);
+  const [selectedStaff, setSelectedStaff] = useState({
+    id: 0,
+    firstName: '',
+    surname: '',
+    jobTitle: '',
+    contractedHours: 0,
+  });
+
+  const closeModal = () => {
+    setDeleteModalState(false);
+  };
+
+  const openDeleteModal = (staff: StaffData) => () => {
+    setDeleteModalState(true);
+    setSelectedStaff(staff);
+  };
+
+  const tableColumns = makeTableTolumns(editUser, openDeleteModal);
```

Comment:
typo, `makeTableTolumns` should have columns

Reply:
'Tolumn' is the official word for 'table column', didn't you know? :stuck_out_tongue_winking_eye: 

fixed